### ISSUE TITLE
Shadow 2.0 support

### ISSUE THREAD

**Issue by ben-manes** [melix/jmh-gradle-plugin#93]
```
* What went wrong:
A problem occurred evaluating script.
> Failed to apply plugin [id 'me.champeau.gradle.jmh']
   > Cannot execute null+configuration ':caffeine:shadow'
  ...
Caused by: java.lang.NullPointerException: Cannot execute null+configuration ':caffeine:shadow'
  ...
  at com.github.jengelman.gradle.plugins.shadow.ShadowJavaPlugin$_apply_closure1.doCall(ShadowJavaPlugin.groovy:35)
```

**Comment by boris-petrov** [comment]
This might be an issue in `shadow`: https://github.com/johnrengelman/shadow/issues/297

**Comment by melix** [comment]
Fixed by #96
